parsec/core/gui/device_recovery_export_widget.py

```diff
@@ -70,7 +80,9 @@ def _print_recovery_key(self):
 class DeviceRecoveryExportPage1Widget(QWidget, Ui_DeviceRecoveryExportPage1Widget):
     info_filled = pyqtSignal(bool)
 
-    def __init__(self, devices, parent=None):
+    def __init__(
+        self, devices: Sequence[LocalDevice | AvailableDevice], parent: Optional[QWidget] = None
```

Comment:
I don't like that. We shouldn't instead change the call in `MainWindow` to provide an `AvailableDevice` instead (there's the function `get_available_device()` in `local_device` for that) and type this as `list[AvailableDevice]`.